Realm/Tests/RealmConfigurationTests.mm

```diff
@@ -96,7 +96,7 @@ - (void)testDefaultConfiguration {
     RLMRealmConfiguration *defaultConfiguration = [RLMRealmConfiguration defaultConfiguration];
     XCTAssertEqualObjects(defaultConfiguration.fileURL, RLMDefaultRealmURL());
     XCTAssertNil(defaultConfiguration.inMemoryIdentifier);
-    XCTAssertNil(defaultConfiguration.encryptionKey);
+    XCTAssertEqual(!!defaultConfiguration.encryptionKey, self.encryptTests);
```

Comment:
Double unary `!`?

Reply:
To convert the pointer value to a boolean, since it should be non-nil only if encryptTests is true.